control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -985,6 +984,14 @@ func (r *HostedControlPlaneReconciler) ensureControlPlane(ctx context.Context, h
 		}
 	}
 
+	registryConfig := manifests.Registry()
+	if _, err := r.CreateOrUpdate(ctx, r.Client, registryConfig, func() error {
+		registry.ReconcileRegistryConfig(registryConfig, hcp.Spec.Platform.Type == hyperv1.NonePlatform)
```

Comment:
I am now seeing there is a `platformType` func that doesn't use the `type` field, is it ok to check that field?

Reply:
that function looks wrong :) I think just checking the Type field should be fine.